### DISCUSSION TITLE
Wiimote constant fall asleep issue

### DISCUSSION THREAD

**Discussion by Sowden** [darthcloud/BlueRetro#988] [General]
Hey there guys.  So I have a question about the Wiimote EXT issue.  I'm using a Retro Rosetta with a Wiimote on my Wii U.  The thing if it is that the Wiimote keeps falling asleep after some minutes of non-use.  When your playing on your Wii, you never notice this cause your Wiimote is in your hands.  You press a button, it connects again, and your off to the races.  And I never normally come against this unless I'm playing a Square Soft game on my PSX emulator, watching one of those long cutscenes.  So I guess what my question is what could be done about this?  Could there be an option for a constant pulse, constant button pressing to keep it alive?  Either on the Wiimote side or the Wii Classic (which is what blueretro emulates) side?  It's a slight annoyance, cause once you jump back into the game with no button inputs, you have to reach to your Wiimote to press a button to turn it back on.  Any thoughts on this?  Thanks.

**Comment by darthcloud** [comment]
It quite a niche problem TBH. I think it's resonable that you need to reach for the Wiimote ;)
